test/configCases/records/issue-2991/test.js

```diff
@@ -0,0 +1,34 @@
+try {
+	require("foo");
+} catch(e){}
+
+it("should write relative paths to records", function() {
+	var fs = require("fs");
+	var path = require("path");
+	var content = fs.readFileSync(path.join(__dirname, "records.json"), "utf-8");
+	content.replace(/\\\\/g, "/").should.eql(`{
+  "modules": {
+    "byIdentifier": {
+      "../../../../external \\"fs\\"": 0,
```

Comment:
Why are there so many `../` in front of this? Looks like there is already something that tries to make it relative, but it looks like that this is broken. see `makeRelative` in Records plugin.